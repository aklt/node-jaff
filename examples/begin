#begin coffee
# Here is some coffee script
foo = __filename
count = 0
bar = ->
  return count += 1
# Change the quotes
__quotes '<', '>'

# Require something
coffee = require 'coffee-script'
#end
Values can be output: <foo> <bar()> <bar()> <bar()>
Using Coffee Script version <coffee.VERSION>
