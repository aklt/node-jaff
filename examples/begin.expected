Values can be output: begin 1 2 3
Using Coffee Script version 1.7.1
